File: Testing/AngularTest/TestApi2/src/main/java/com/AngularTest/TestApi2/data/PersonRepository.java
package com.AngularTest.TestApi2.data;

import org.springframework.data.jpa.repository.JpaRepository;
import com.AngularTest.TestApi2.domain.models.Person;
import com.AngularTest.TestApi2.intent.IntentMerge;
import java.util.UUID;

/**
 * Spring Data JPA repository for the Person entity.
 */
@IntentMerge
public interface PersonRepository extends JpaRepository<Person, UUID> {
}

File: Testing/AngularTest/TestApi3/src/repository/person.repository.ts
import { Repository } from 'typeorm';
import { CustomRepository } from './../typeorm/typeorm-ex.decorator';
import { Person } from './../domain/entities/person.entity';
import { IntentMerge } from './../intent/intent.decorators';

@IntentMerge()
@CustomRepository(Person)
export class PersonRepository extends Repository<Person> {
}